.vscode/settings.json

```diff
@@ -18,6 +18,7 @@
 	"search.exclude": {
 		"**/node_modules": true,
 		"**/bower_components": true,
+		"cli/target/**": true,
```

Comment:
small unrelated fix: I noticed search was getting slow, which I assumed was Defender, but it was actually looking through all the build output in the CLI